app/app_test.go

```diff
@@ -51,7 +51,6 @@ func Server() *martini.ClassicMartini {
 	m.Get("/v1/space/:space/apps", Describespace)              //describeapp.go
 	m.Get("/v1/space/:space/app/:appname", DescribeappInSpace) //describeapp.go
 
-	m.Get("/v1/space/:space/app/:appname/deployments", GetDeployments) //replicasets.go
 	m.Post("/v1/space/:space/app/:appname/restart", Restart)           //restart.go
```

Comment:
Looked through logs and controller/taas code couldn't find anywhere this was used, pulling it.